src/dispatch/plugins/dispatch_jira/plugin.py

```diff
@@ -222,6 +223,7 @@ def update(
         storage_weblink: str = None,
         labels: List[str] = None,
         cost: int = None,
+        incident_type_plugin_metadata: dict = {},
```

Comment:
Is this still a WIP? Looks like we haven't let Jira read the incident metadata.

Reply:
Can port generic Jira plugin in another PR?

Reply:
Separate PR is fine.